tools/modulate.py

```diff
@@ -32,6 +32,9 @@ def main():
     parser.add_argument("--install-name-tool", metavar="/path/to/install_name_tool", type=str, default=None)
     parser.add_argument("--strip", metavar="/path/to/strip", type=str, default=None)
 
+    the_endians = ('big', 'little')
+    parser.add_argument('--endian',  metavar=("|".join(the_endians)), type=str, default='little', choices=the_endians)
```

Comment:
Strings should be double-quoted, except for enum values, which should be single-quoted. (So `'big'` and `'little'` are correct.)

Reply:
Is this the change you meant? Did I make the change correctly? Not an expert with github!